src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/conf/ReadYamlStep.java

```diff
@@ -165,6 +95,8 @@ public Step newInstance(Map<String, Object> arguments) throws Exception {
 		 */
 		@Override
 		protected Object run() throws Exception {
+			super.run();
```

Comment:
Another, perhaps better approach is to have another `abstract` method in `AbstractFileOrTextStepExecution` that is called from it's `run` method and then override that abstract method here. That way you don't forget to call `super.run();` in every sub class.